lib/modules/platform/gitlab/index.spec.ts

```diff
@@ -2539,6 +2539,40 @@ describe('modules/platform/gitlab/index', () => {
         })
       ).toResolve();
     });
+
+    it('adds and removes labels', async () => {
+      await initPlatform('13.3.6-ee');
+      httpMock
+        .scope(gitlabApiHost)
+        .get(
+          '/api/v4/projects/undefined/merge_requests?per_page=100&scope=created_by_me'
+        )
+        .reply(200, [
+          {
+            iid: 1,
+            source_branch: 'branch-a',
+            title: 'branch a pr',
+            state: 'open',
+            description: `<!--labels:WyJvbGRfbGFiZWwiXQ==-->`, // equivallent to labels array : ['old_label']
+          },
+        ])
+        .put('/api/v4/projects/undefined/merge_requests/1')
+        .reply(200);
+      await expect(
+        gitlab.updatePr({
+          number: 1,
+          prTitle: 'title',
+          prBody: 'body',
+          state: 'closed',
+          addLabels: ['new_label'],
+          removeLabels: ['old_label'],
+        })
+      ).toResolve();
+      expect(logger.debug).toHaveBeenCalledWith([
+        `Assigning labels new_label to #1`,
+        `Unassigning labels old_label from #1`,
```

Comment:
_Unassign_ is a clunky verb. We should put our points positively.

The GitHub Docs uses _apply_ and _dismiss_. [^1] I like those verbs a lot. To summarize:

| Bad verb | Good verb | 
| --- | --- |
| Assign | Apply |
| Unassign | Dismiss |

[^1]: [GitHub Docs, applying a label](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels#applying-a-label)

Reply:
I was trying to use the terminology used by each platform as found in the API. 

I am still not sure whether to log this or not @rarkins WDYT??